cypress/integration/addOffer.spec.js

```diff
@@ -0,0 +1,47 @@
+// <reference types="Cypress"/>
+
+// Example import
+// import * as Texts from 'constants/objectTexts'
+const todayDate = Cypress.moment().format('MMMM DD, YYYY')
+
+describe('Login', function() {
+  beforeEach(function() {
+    cy.visit('/')
+  })
+  it('Login passed', () => {
+    cy.get('input[id="email-login"]').type('b2ctwenty@gmail.com')
+    cy.get('input[id="password-login"]').type('123456')
+    cy.get('button[id="login-btn"]').click()
+    cy.contains('li', 'Arina')
+  })
+  it('Add offer', () => {
+    cy.contains('a', 'Grundstücksdatenbank').wait(2000)
+    cy.get('#mkl').click()
+
+    cy.contains('a', 'GRUNDSTÜCKE HINZUFÜGEN')
+    cy.get('[id="New Offer"]')
+      .click()
+      .wait(2000)
+
+    cy.get('input')
+      .eq(1)
+      .type('Friedrichstraße Test')
+
+    cy.get('[name="date-time"]')
+      .click()
+      .wait(1000)
+    cy.get(`[aria-label="${todayDate}"]`).click()
```

Comment:
ตอนเรียกใช้อาจจะเรียกเป็นแบบนี้ครับ ถ้าเอาไปเพิ่มแล้ว
```suggestion
    cy.get(`[aria-label="${Texts.todayDate}"]`).click()
```